### PR TITLE
Gp 21 fe landing page

### PR DESCRIPTION
**Pull request by thuanvmse161224**

### CODE REVIEW

File: src/pages/Landing/Contact/ContactForm/components/EmailTextField.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import ContactFormTextField from './ContactFormTextField'
+
+
+type EmailTextFieldProps = {
+	errors: FieldErrors<ContactFormType>,
+	register: UseFormRegister<ContactFormType>,
+};
+
+// eslint-disable-next-line no-useless-escape
+const EMAIL_REGEX = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
```
Comment (danielNg91) [inline src/pages/Landing/Contact/ContactForm/components/EmailTextField.tsx]: this regex can be reused, put it in common or contants.ts and import it

File: src/pages/Landing/Landing.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+const ButtonStyle2 = {
+	width: 'calc(45% - 20px)', padding: '10px', margin: '10px', borderRadius: '25px', height: '50px', backgroundImage: 'linear-gradient(135deg, #7F23F5, #2824F5)',
+}
+
+
+export const Landing: React.FC = () => {
+	return (
+		<>
+			<div style={{ backgroundColor: '#39289F', padding: '5vw'}} id='landing-main'>
```
Comment (danielNg91) [inline src/pages/Landing/Landing.tsx]: with lots of styles here, it's  better to be create a styled component for this div

**Review by danielNg91 — CHANGES_REQUESTED** [review]

File: src/pages/Landing/Contact/ContactForm/components/EmailTextField.tsx (hunk 1 above)
Comment (thuanvmse161224) [reply src/pages/Landing/Contact/ContactForm/components/EmailTextField.tsx]: Done

File: src/pages/Landing/Landing.tsx (hunk 2 above)
Comment (thuanvmse161224) [reply src/pages/Landing/Landing.tsx]: done

**Review by danielNg91 — APPROVED** [review]

**Pull request merged**